### PR TITLE
Blog Post - Ipython Notebook

### LINKED ISSUE #112: Blog Post of Density Data
Hey 

Attached is a link to the notebook with my plots I was gonna share at devfest.

Let me know what edits you think I should make before we put it up on the blog:

http://nbviewer.ipython.org/github/jzf2101/density/blob/master/datasci/density_demo.ipynb

### PR DESCRIPTION
**Pull request by jzf2101**
Closes #112 See discussion and notebook. 

Notebook link [here](http://nbviewer.ipython.org/github/jzf2101/density/blob/master/datasci/density_demo.ipynb) to preview

### CODE REVIEW

**Comment by natebrennand** [comment]
are we just linking to the iPython notebook or making an HTML page hosting this?

File: datasci/density_blog.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+##And Orgo Night!
+![alt text](orgonight.png)
+
+Notice the spike in Butler 2 around midnight.
+
+##For more tools with the Density API, check out Chris Mulligan's [Relative Density App!](https://chmullig.shinyapps.io/relative_density/)
+
+
+#To see the code that generated check out our [ipython notebook!](http://nbviewer.ipython.org/github/jzf2101/density/blob/master/datasci/density_demo.ipynb)
```
Comment (thebrianzeng) [inline datasci/density_blog.md]: "code that generated all of our visuals" or something of the sort

**Comment by thebrianzeng** [comment]
we can put the markdown as a blog post on adicu.com/blog and then link to
both in the advertising.

File: datasci/density_blog.md (hunk 1 above)
Comment (jzf2101) [inline datasci/density_blog.md]: Good idea.

**Comment by jzf2101** [comment]
@thebrianzeng I think using the markdown file as a blog post is great idea.  Could someone show me how we post to the blog? It'd take extra work for me to figure out how to host the notebook on our own server, but we could do that if we want to.

**Comment by jzf2101** [comment]
ok so #133 seems to be related to the changes I made in this pull request, so I think I'll need to rework it so it can still run

**Comment by natebrennand** [comment]
There are no changes to any of the python files in the PR. Are there changes that you have not pushed to this branch?

**Comment by jzf2101** [comment]
nvm that has nothing to do with it.

**Comment by jzf2101** [comment]
@thebrianzeng Updated the language with your suggestions.  Could you look again and confirm it's cool with you?

**Comment by evantarrh** [comment]
:+1:

**Pull request merged**